index.md

```diff
@@ -91,6 +91,16 @@
 | **Cluster version** | 0.12.1 |
 | **Hosted by** | [Temporal.Cloud](https://temporal.cloud) |
 
+---
+
+| | Wikipedia |
+| - | - | 
+| **Description** | Snapshots of Wikipedia articles for various languages |
+| ***Init&Run*** | `ipfs-cluster-follow ipfs-search run --init ipfs-collab.wikipedia` |
```

Comment:
`ipfs-collab.wikipedia` seems off. This needs to be a resolvable domain name with dnslink setup (we do the equivalent of a `wget https://ipfs.io/ipns/<name>`.

I think our scripts set this up automatically given the cluster name, so perhaps `wikipedia.collab.ipfscluster.io` might be the right thing (depending on how you named the cluster).